tests/test_repositoryArgs.py

```diff
@@ -122,12 +120,12 @@ def testAbsoluteFilePathWithSchema(self):
         uri = urllib.parse.urljoin('file://', self.testDir)
         args = dp.RepositoryArgs(root=uri, mapper='lsst.daf.persistence.Mapper')
         butler = dp.Butler(outputs=args)
+        self.assertIsInstance(butler, dp.Butler)
         self.assertTrue(os.path.exists(os.path.join(self.testDir, 'repositoryCfg.yaml')))
 
-        butler2 = dp.Butler(inputs=uri)
```

Comment:
I removed this since it seemed to be the same call as the next one but not protected by a try block.